Dockerfile

```diff
@@ -0,0 +1,32 @@
+FROM node:5.11-wheezy
+
+ARG admin_user
+
+WORKDIR /opt/ldnode/certs
+RUN openssl genrsa 2048 > ssl-key.pem
+RUN openssl req -new -x509 -nodes -sha256 -days 3650 -key ssl-key.pem -subj '/CN=*.localhost' > ssl-cert.pem
```

Comment:
This line will just generate default certs for the container to use in the absence of "proper" certs; you load them in with a volume mount at run time (as part of your `docker run ...` command). This is documented on line 83 of `README.md`.